bigtest/tests/provider-edit-test.js

```diff
@@ -0,0 +1,99 @@
+import { expect } from 'chai';
+import { describe, beforeEach, it } from '@bigtest/mocha';
+
+import { describeApplication } from '../helpers/describe-application';
+import ProviderShowPage from '../interactors/provider-show';
+import ProviderEditPage from '../interactors/provider-edit';
+
+describeApplication('ProviderEdit', () => {
+  let provider,
+    packages;
+
+  beforeEach(function () {
+    provider = this.server.create('provider', 'withPackagesAndTitles', 'withProxy', {
```

Comment:
Not sure if this param being passed is serving any specific purpose - but should `withPackagesAndTitles` be renamed to `withPackages` or is it legit to say `withPackagesAndTitles`?

Reply:
It is legit to say  `withPackagesAndTitles`  in fact there is not a trait on Providers for just `withPackages`

Reply:
ok 👍